src/reducers/entities.js

```diff
@@ -13,24 +13,24 @@ const reducer = handleActions({
     [action.payload.entityType]: {
       ...state[action.payload.entityType],
       [action.payload.data.id]: {
-        ...get(state, `[${action.payload.entityType}][${action.payload.data.id}]`, {}),
+        ...get(Object.assign({}, state), `[${action.payload.entityType}][${action.payload.data.id}]`, {}),
         ...action.payload.data,
       },
     },
   }),
   [receiveEntities]: (state, action) => {
-    const result = { ...state }
+    const result = { ...(Object.assign({}, state)) }
     keys(action.payload.entities).forEach((entityType) => {
       // preserve old entities
-      const entitiesState = get(state, `${entityType}`, {})
+      const entitiesState = { ...(get(state, `${entityType}`, {})) }
       set(result, `${entityType}`, entitiesState)
 
       keys(action.payload.entities[entityType]).forEach((entityId) => {
         // preserve old entity (only update if not exists)
-        const entityState = get(state, `${entityType}.${entityId}`, {})
+        const entityState = get(Object.assign({}, state), `${entityType}.${entityId}`, {})
         set(result, `${entityType}.${entityId}`, {
           ...entityState,
-          ...action.payload.entities[entityType][entityId],
+          ...Object.assign({}, action.payload.entities[entityType][entityId]),
```

Comment:
Why are you using the spread operator before ```Object.assign```?

Reply:
Yeah, I've the same question. I think we can get rid of it here. Or maybe it will be fixed after the backmerge is done?

Reply:
```
var _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; };
```
This is a polyfill method from babel which is used to transpile spread operator. So you are doing `Object.assign({}, Object.assign({}, action.payload.entities[entityType][entityId]))`

Reply:
Can you explain to me how I can use `Object.assign` without spreading it? By putting everything into a variable first and then doing Object.assign on it?

Reply:
https://redux.js.org/docs/recipes/UsingObjectSpreadOperator.html

If it is not clear enough, let me know and I will provide you more concrete examples.